canonical-web-and-design/ubuntu.com
# Exclude tag not:ubuntu on ubuntu.com/blog

## Done

**Before merging please warn me so I remove the tag!**

- Exclude tag `not:ubuntu` that has the id `3960` (you can view [here](https://admin.insights.ubuntu.com/wp-admin/term.php?taxonomy=post_tag&tag_ID=3960&post_type=post&wp_http_referer=%2Fwp-admin%2Fedit.tags.php%3Ftaxonomy%3Dpost_tag%26post_type%3Dpost%26s%3Dnot%253Aubuntu) VPN only, sorry external)
- This will allow other blogs that need to post articles that shouldn't appear on ubuntu.com/blog For example a demonstration of snap on another distro

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/blog
- should work as usual.
- Try to access the article: https://ubuntu-com-8379.demos.haus/blog/stepping-down-gracefully
- Should 404 (I added  a temporary tag)
- It shouldn't appear in the blog list too https://ubuntu.com/blog
